2021-Greenwald_Miller_et_al-Mesmer/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/angelolab/ark-analysis.git@6b21140e099ce21a458699573e8dbda9127c3d03
```

Comment:
eww

Reply:
Hahahaha, avert your eyes! You weren't supposed to see this. I'll make a new release this weekend.